upup/pkg/fi/cloudup/containerd.go

```diff
@@ -35,7 +35,7 @@ const (
 	// containerd legacy packages URLs for v1.2.x and v1.3.x
 	containerdLegacyUrlAmd64 = "https://storage.googleapis.com/cri-containerd-release/cri-containerd-%s.linux-amd64.tar.gz"
 	// containerd version that is available for both AMD64 and ARM64, used in case the selected version is not available for ARM64
-	containerdFallbackVersion = "1.4.3"
+	containerdFallbackVersion = "1.4.4"
```

Comment:
Please revert here, at the moment there is not 1.4.4 version for ARM64, so the fallback remains 1.4.3.

Reply:
There are `v1.4.4` rpms available for CentOS 7/8 and Ubuntu 20.10 I believe, but it's not widely available, so happy to keep the fallback version as `v1.4.3`.

Reply:
kOps uses static binaries, not the rpm/deb. Unfortunately we can't do much until the next release.